app/controllers/dashboard/goals_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Dashboard
+  class GoalsController < ApplicationController
+    skip_before_action :require_authentication
+
+    def new
+      @goal = Goal.new
```

Comment:
Acho que que seria legal se a data atual já aparecesse preenchida com a data de hoje no form:

```suggestion
      @goal = Goal.new(start_date: Time.zone.today)
```